JavaFXUserInterface/src/main/java/controllers/AssetsController.java

```diff
@@ -342,14 +429,16 @@ public void generateList() {
         descriptionCol.setCellValueFactory(
                 new PropertyValueFactory<>("description"));
 
-        table.setItems(assets);
-        table.setId("listTable");
-        table.getColumns().addAll(assetTypeCol, serialNoCol, modelCol, modelRULCol, recommendationCol, locationCol, siteCol, categoryCol, manufacturerCol, descriptionCol);
-        AnchorPane.setBottomAnchor(table, 0.0);
-        AnchorPane.setTopAnchor(table, 5.0);
-        AnchorPane.setRightAnchor(table, 0.0);
-        AnchorPane.setLeftAnchor(table, 0.0);
-        assetsListPane.getChildren().addAll(table);
+        if (assetsDisplayed != null) {
```

Comment:
I think it would be good to provide an alternative text to display on the page in the case where there are no assets to display. What I had in mind would be kind of a grey text appearing in the middle of the screen stating that there are "No results for your search" or something like that.

Reply:
Table now shows "No results found"

Reply:
Nice touch on the no results found.